src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/RazorSyntaxTreeExtensions.cs

```diff
@@ -41,5 +41,17 @@ public static RazorDirectiveSyntax[] GetCodeBlockDirectives(this RazorSyntaxTree
 
             return codeBlockDirectives;
         }
+
+        public static CSharpStatementSyntax[] GetCSharpStatements(this RazorSyntaxTree syntaxTree)
```

Comment:
Any reason to not return an IReadOnlyList here (arrays are IReadOnlyLists)?

Reply:
No reason. Can do.